client/components/forms/form-setting-explanation/index.tsx

```diff
@@ -0,0 +1,18 @@
+import clsx from 'clsx';
+import { forwardRef } from 'react';
+
+import './style.scss';
```

Comment:
Side note and a potential follow-up:

I saw a single use case for this `button.is-borderless` in the `FormSettingExplanation` stylesheet here:

https://github.com/Automattic/wp-calypso/blob/0244a1ecae85bbe4e1bef053f4dc23543bbd8fe5/client/my-sites/site-settings/form-general.jsx#L336-L357

With that in mind, can/should that be moved away of the component and inlined into the site settings stylesheet?

Reply:
Absolutely, the `button` styles in the `FormSettingExplanation` are breaking style encapsulation 😓

Reply:
Working on this in https://github.com/Automattic/wp-calypso/pull/98887